packages/merge-styles/src/styleToClassName.ts

```diff
@@ -0,0 +1,179 @@
+import { GlobalSettings } from '@uifabric/utilities/lib/GlobalSettings';
+import { rtlifyRules } from './transforms/rtlifyRules';
+import { provideUnits } from './transforms/provideUnits';
+import { prefixRules } from './transforms/prefixRules';
+import { expandQuads } from './transforms/expandQuads';
+import { getVendorSettings } from './getVendorSettings';
+import { Stylesheet } from './Stylesheet';
+import { IStyle } from './IStyle';
+
+const DISPLAY_NAME = 'displayName';
+
+function toRuleEntry(name: string, value: string): string | undefined {
+  if (name === DISPLAY_NAME) {
+    return undefined;
+  }
+
+  // Prefixes like WebKit need a dash infront of them.
+  if (/[A-Z]/.test(name[0])) {
+    name = '-' + name;
+  }
+
+  // Kebab case the rule.
+  let rules: string[] = [
```

Comment:
Shouldn't we use an actual type instead of the `+= 2` logic everywhere we touch `rulePairs`? 
(if not, rename `rules` to `rulePairs` here?)